Makefile

```diff
@@ -25,7 +25,7 @@ endif
 
 ifndef NOCOLOR
   GREEN  := $(shell tput -Txterm setaf 2)
-  YELLOW := $(shell tput -Txterm setaf 3)
+  YELLOW := $(shell tput -Txterm seaf 3)
```

Comment:
does not make sense to me?

Reply:
sorry, my bad, that was a coding while driving with the bus through a bumpy mountain fuckup - should have checked the changes more properly. Just resolved this